### ISSUE TITLE
Hi I've run 10+ epochs,  but the training loss still changes between 2~3 and BLEU score is less than 9 

### ISSUE THREAD

**Issue by Anothernewcomer** [microsoft/CodeBERT#95]
![1639580679(1)](https://user-images.githubusercontent.com/38857893/146211020-e7223b76-8478-469c-9d18-57a1ef632a15.png)
And as you can see , the BLEU score is also low after more than 10 epochs of training..I use google colab so I had to train from ckpt once in a while that's why the log file says it's epuch No.1 but it's actually alredy 12 epochs
I only  altered the batch_size to 16 (because of GPU limits) as well as the data process part (to read files and convert examples to features  offline and read features online )
I can't figure out why i got this, help me please

**Comment by guoday** [comment]
I don't know what task you do. Can you please give more details. What task do you do? By the way, our code doesn't support re-training from a ckpt. Besides, `Best ppl` should be about 10 but you get 59120. Maybe you need to check whether you really follow the Readme. From the result, `the data process part` for test/dev dataset may be wrong.

**Comment by Anothernewcomer** [comment]
Yes I use the original code and re-ran, the result so far seemed totally fine so it  has to be bugs in my data process part .Thanks  a lot!
